src/doc/performance-tuning.md

```diff
@@ -257,6 +439,71 @@ root@c819e0f106c4:/# taskset -cp $$
 pid 1's current affinity list: 0-15
 ```
 
+## Application-specific performance notes
+
+### lwAFTR
+
+#### Avoid fragmentation
+
+Fragment reassembly is a costly operation.  Make sure that network
+MTUs are set such that fragmentation is rare.
+
```

Comment:
As this is a PR for master which doesn't have these knobs yet afaiu, I'll leave these out for now.  But yes we should doc them!